go/tasks/logs/config.go

```diff
@@ -1,21 +1,43 @@
 package logs
 
-import "github.com/lyft/flyteplugins/go/tasks/config"
+import (
```

Comment:
there is an example config.yaml. Do you want to add this to the example?

Reply:
I mean add an example for the logging?

Reply:
Done